src/key_exchange/group/curve25519.rs

```diff
@@ -57,9 +57,9 @@ impl KeGroup for Curve25519 {
 
     // Implements the `HashToScalar()` function from
     // <https://www.ietf.org/archive/id/draft-irtf-cfrg-voprf-09.html#section-4.1>
```

Comment:
Do you mind updating this (and other occurrences of) pointer to the version 19 draft of VOPRF?

Reply:
On it.

Reply:
Done.